### PR TITLE
SAK-44571 Lessons: Embedded hidden PDF resources are visible to students

### PR DESCRIPTION
**Pull request by MRutea**

### CODE REVIEW

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java [hunk 1]
```diff
@@ -1261,7 +1261,15 @@ public void printSubpage(List<SimplePageItem> itemList, boolean first, UIBranchC
 			boolean forceButtonColor = false;
 			String color = null;
 			for (SimplePageItem i : itemList) {
-
+				
+				//If the content MULTIMEDIA, type 7, is not released or is hidden will not be rendered on the page
+				if(i.getType()==SimplePageItem.MULTIMEDIA) {
```
Comment (adrianfish) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java:1266]: Need some formatting here. Spaces after the if and spaces around the ==. It just makes it a bit easier to read.

**Review by adrianfish — CHANGES_REQUESTED** [review]

**Comment by adrianfish** [comment]
I'd just clean this up now. The commit message isn't right, probably because of the rebasing you've been doing. I'd just squash it into one commit and make sure it's clean. I'll approve it then. Thanks for indulging me :)

**Comment by MRutea** [comment]
Hi, Adrian;
Thank you.
I've just ammended the commit message.

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java [hunk 2]
```diff
@@ -1262,6 +1262,14 @@ public void printSubpage(List<SimplePageItem> itemList, boolean first, UIBranchC
 			String color = null;
 			for (SimplePageItem i : itemList) {
 
+				//If the content MULTIMEDIA, type 7, is not released or is hidden will not be rendered on the view
+				if (i.getType() == SimplePageItem.MULTIMEDIA) {
+				    boolean isContentAvailable = contentHostingService.isAvailable(String.valueOf(i.getSakaiId()));
+				    if (!isContentAvailable) {
+				        continue;
+				    }
```
Comment (adrianfish) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java:1270]: One more small thing: Just remove the boolean variable assignment. It doesn't seem to be used anywhere else, so it's not needed.

if (!contentHostingService.isAvailable(String.valueOf(i.getSakaiId()))) {
    continue;
}

**Review by adrianfish — CHANGES_REQUESTED** [review]

**Comment by MRutea** [comment]
@adrianfish 
Hi, Adrian;
Done!
Thank you.

**Review by adrianfish — APPROVED** [review]
Good to go! Thanks :)

**Pull request merged**